cms/envs/test.py

```diff
@@ -51,28 +54,26 @@
 STUDIO_SHORT_NAME = gettext_lazy("𝓢𝓽𝓾𝓭𝓲𝓸")
 
 # Allow all hosts during tests, we use a lot of different ones all over the codebase.
-ALLOWED_HOSTS = [
-    '*'
-]
+ALLOWED_HOSTS = ["*"]
```

Comment:
Aargh, I didn't mean to let my IDE reformat this file to our current standards. I mean, it's not wrong, but it makes life harder for the reviewers. Sorry. 😬 I think it's just this file, though.